modules/configure-cvo-retrieve-image-signatures.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the local OSUS Operator.
+This configuration is done by adding the URI of OSUS Operator's signature serving endpoint to the CVO's list of signature stores.
+
+This is a one-time configuration that prevents the need to apply release image signatures each time you mirror a new set of release images.
```

Comment:
Maybe "prevents" -> "avoids"?  Not entirely sure why I feel that way; maybe I expect "prevents" for bad things and "avoids" for annoying things, and per-update signature application is annoying, but not bad?  Feel free to ignore if this makes no sense :)

Reply:
I'm not super opinionated so "avoids" would be fine by me! The logic makes sense to me, personally